cmd/teleirc.go

```diff
@@ -24,36 +24,35 @@ var (
 func main() {
 	flag.Parse()
 
-	verbose := internal.Debug { DebugLevel:  *flagDebug }
+	debug := internal.Debug { DebugLevel:  *flagDebug }
 
 	if *flagVersion {
-		verbose.PrintVersion("Current TeleIRC version: " + version)
+		debug.PrintVersion("Current TeleIRC version: " + version)
 		return
 	}
 
-	// TODO: Build out debugging capabilities for more verbose output
 	// Notify that debug is enabled
-	verbose.LogInfo("Debug mode enabled!")
+	debug.LogInfo("Debug mode enabled!")
 
 	settings, err := internal.LoadConfig(*flagPath)
 	if err != nil {
-		verbose.LogError(err)
+		debug.LogError(err)
 		os.Exit(1)
 	}
 
 	var tgapi *tgbotapi.BotAPI
-	tgClient := tg.NewClient(settings.Telegram, tgapi, verbose)
-	tgChan := make(chan error)
+	tgClient := tg.NewClient(settings.Telegram, tgapi, debug)
+	tgChan   := make(chan error)
```

Comment:
The `:=` on line 45 doesn't need to be in line with line 44. Padding is typically stylistically done when there are 3-4+ variables all being instantiated at the same time.